Package.swift

```diff
@@ -201,7 +201,7 @@ let package = Package(
             name: "Basics",
             dependencies: [
                 "_AsyncFileSystem",
-                .target(name: "SPMSQLite3", condition: .when(platforms: [.macOS, .iOS, .tvOS, .watchOS, .visionOS, .macCatalyst, .linux])),
+                .target(name: "SPMSQLite3", condition: .when(platforms: [.macOS, .iOS, .tvOS, .watchOS, .visionOS, .macCatalyst, .linux, .custom("freebsd")])),
                 .product(name: "SwiftToolchainCSQLite", package: "swift-toolchain-sqlite", condition: .when(platforms: [.windows, .android])),
```

Comment:
Great to see this port moving forward. 🥲

Have you tried using this SQLite package instead? The only reason most OSs above don't is that this is new: I switched my upcoming native 6.1 toolchain for Android over to using it and was able to [remove the external libsqlite package dependency](https://github.com/finagolfin/termux-packages/commit/a3d06c314b0e77e026cbcb19c58883b19d85af0d#diff-a3bcec5381cf1ae076d0c16e6be6792a91869d20be55c919085f0704fa11424cL11) of this Swift toolchain as a result. I suggest you try the same, will require [patching llbuild also](https://github.com/swiftlang/swift-llbuild/pull/954/files#diff-f913940c58e8744a2af1c68b909bb6383e49007e6c5a12fb03104a9006ae677eR17).

Reply:
Ah, I just saw your comment. This sounds like a good idea! Thanks!